pkg/synthetictests/networking.go

```diff
@@ -15,7 +17,7 @@ type testCategorizer struct {
 	substring string
 }
 
-func testPodSandboxCreation(events monitorapi.Intervals) []*ginkgo.JUnitTestCase {
+func testPodSandboxCreation(unsortedEvents monitorapi.Intervals) []*ginkgo.JUnitTestCase {
```

Comment:
Wait, these should be sorted by default.  Are they not?

Reply:
(I thought they would be sorted)

Reply:
I don't think these are sorted.
We're reading those in [`EventsFromFile`](https://github.com/openshift/origin/blob/2b30f8e6a2a55ea3f7f1415ba8fdcf5135888c4b/pkg/monitor/serialization/serialize.go#L38) and written in [`EventsToJSON`](https://github.com/openshift/origin/blob/2b30f8e6a2a55ea3f7f1415ba8fdcf5135888c4b/pkg/monitor/serialization/serialize.go#L67-L76), neither is explicitely sorting events.

In any case, if these are sorted, it would take a constant time to sort them, as we do it once when running the test

Reply:
Ah, we do [sort](https://github.com/openshift/origin/blob/2b30f8e6a2a55ea3f7f1415ba8fdcf5135888c4b/pkg/monitor/serialization/serialize.go#L73) them on write, my bad.

Removing sorting in the test